### PR TITLE
LPD-30653 Create checkbox on object action to update the usePreferredLanguageForGuests parameter

### PR DESCRIPTION
**Pull request by igor-franca**
The acceptance criteria can be viewed [here](https://liferay.atlassian.net/browse/LPD-28487). If you find anything that doesn't comply with the criteria, let me know.
Test case [spreadsheet](https://docs.google.com/spreadsheets/d/1Cpk1A_lbD00q6ejHfvqLXqIe8CBubZ2-XtCD5Wbc05I/edit?usp=sharing)

Main contributions:

In contact with Julia, we agreed to move the `relatedObjectEntries` checkbox to the same position proposed for the `usePreferredLanguageForGuests` checkbox. In order to maintain consistency.

To perform this test you can relate two objects and choose "Add Object Entry" in the Then field.

move-checkbox

Furthermore, a new checkbox was added to update the `usePreferredLanguageForGuests` parameter.

Which can be viewed by creating an "On After Add" or "On After Update" trigger and selecting the "Notification" in Then fiied. Finally, we must choose a previously created email notification template.

screenshot-1720722494

### CODE REVIEW

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
I noticed some inconsistencies with the` ant format-source-current-branch` formatter.

But after performing a rebase with the most recent master, the formatter made some formatting changes to my code.

The next push aims to send these formatting changes:

**Comment by igor-franca** [comment]
ci:test:sf

File: modules/test/playwright/tests/object-web/objectActions.spec.ts [hunk 1]
```diff
@@ -152,6 +152,64 @@ test.describe('Manage object actions through object actions tab', () => {
 			await expect(page.getByText(objectAction)).toBeVisible();
 		}
 	});
+
+	test('can create an email notification object action using user preferred language', async ({
```
[the remark this answers is not in the document]
Comment (igor-franca) [reply modules/test/playwright/tests/object-web/objectActions.spec.ts:156]: Hey Carlos, that's a valid concern, thanks for bringing it up. The BE tests have not yet been added to the spreadsheet, but they can be viewed in the [refinement document](https://liferay.atlassian.net/wiki/spaces/DXPL/pages/2981691408/LPD-28487+Allow+Admin+Users+to+Configure+Email+Language+for+Guests). 

Checks related to email language are being carried out in BE tests.

**Comment by igor-franca** [comment]
Hey Carlos, I did the rename purpose.

Furthermore, I ended up realizing that I needed to make another change, I parameterized the disabled property in the checkbox component.

You can view the changes made [here](https://github.com/liferay-objects/liferay-portal/compare/fb1f5702d6b394ca68960599e5d12de47c3ccffa..924d165e016e67457cb8b645034ae0311b950d9e)

**Comment by igor-franca** [comment]
ci:test:sf

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:test:playwright

**Comment by igor-franca** [comment]
ci:forward

**Pull request closed without merging**